models/pull.go

```diff
@@ -57,6 +69,7 @@ type PullRequest struct {
 	BaseRepo        *Repository `xorm:"-"`
 	HeadBranch      string
 	BaseBranch      string
+	TopicBranch     string           // use for agit style pr
```

Comment:
Could we still use column `HeadBranch` since we have `Style` column, we can know `HeadBranch` is `TopicBranch`?